README.md

```diff
@@ -56,7 +56,7 @@ Make sure you activated the full features of the tokio crate on Cargo.toml:
 
 ```toml
 [dependencies]
-tokio = { version = "1.18.0", features = ["full"] }
+tokio = { version = "1.18.1", features = ["full"] }
```

Comment:
do we actually need to have this be .1 or can it be `1.18`?

Reply:
The CI check wont accept it as it is currently written.